website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -120,20 +120,27 @@ If TLS is enabled on Consul, you will also need to add the following environment
 
 ## Bootstrap Configuration
 
-Envoy requires an initial bootstrap configuration file. The easiest way to
-create this is using the [`consul connect envoy`
-command](/commands/connect/envoy). The command can either output the
-bootstrap configuration directly to stdout, or generate the configuration and issue an `exec` command
-to the Envoy binary as a convenience wrapper. For more information about using `exec` to bootstrap Envoy, refer to [Exec Security Details](/consul/commands/connect/envoy#exec-security-details).
+Envoy requires an initial bootstrap configuration file that is created in one of the two following
+ways:
+
+- When using a Consul Client agent, the easiest way to create the Envoy bootstrap configuration is
+  using the [`consul connect envoy` command](/commands/connect/envoy). The command can either output
+  the bootstrap configuration directly to stdout, or generate the configuration and issue an `exec`
+  command to the Envoy binary as a convenience wrapper. For more information about using `exec` to
+  bootstrap Envoy, refer to [Exec Security
+  Details](/consul/commands/connect/envoy#exec-security-details).
```

Comment:
```suggestion
### Generate the bootstrap file on the Consul CLI

Connect to a local Consul client agent and run the [`consul connect envoy` command](/commands/connect/envoy) to create the Envoy bootstrap configuration. The command either outputs the bootstrap configuration directly to stdout or generates the configuration and issues an `exec` command to the Envoy binary as a convenience wrapper. For more information about using `exec` to bootstrap Envoy, refer to [Exec Security Details](/consul/commands/connect/envoy#exec-security-details).
```
Ignore if you disagree, but I think these work better as subtopics than as bullet items because 1) it makes for a fairly large bullet and 2) it segments the information a little more clearly.